src/components/__tests__/LibrariesList-test.tsx

```diff
@@ -43,56 +43,49 @@ describe("LibrariesList", () => {
           "registry_stage": "cancelled"
         }
       }
-    ]
-  };
-  let fetchData: Sinon.SinonStub;
+    ];
+
   let wrapper: Enzyme.CommonWrapper<any, any, {}>;
   let store;
   describe("rendering", () => {
-    fetchData = Sinon.stub();
+
     beforeEach(() => {
       store = buildStore();
       wrapper = Enzyme.mount(
         <LibrariesList
-          fetchData={fetchData}
           store={store}
           libraries={libraries}
         />
       );
     });
 
-    it("should call fetchData on load", () => {
-      expect(fetchData.callCount).to.equal(1);
-    });
-
     it("should display a list of libraries", () => {
-      let libraryItems = wrapper.find(".panel");
+      let libraryItems = wrapper.find(".panel").not(".panel-info");
       expect(libraryItems.length).to.equal(2);
     });
 
     it("should render each library's  name and short name", () => {
-      let lib1 = wrapper.find(".panel").at(0);
+      let lib1 = wrapper.find(".panel").not(".panel-info").at(0);
       expect(lib1.text()).to.contain("Test Library 1 (lib1)");
-      let lib2 = wrapper.find(".panel").at(1);
+      let lib2 = wrapper.find(".panel").not(".panel-info").at(1);
```

Comment:
The `.not` negation to get an element is weird. Is there a way to get it without having to call `not`? But not a big deal.